src/pages/List/components/Sort.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useSearchParams } from 'react-router-dom';
+import './Sort.scss';
+
+const Sort = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const FILTER = [
+    { id: 1, text: '전체', value: '' },
+    { id: 2, text: '최신순', value: 'newest' },
+    { id: 3, text: '인기순', value: 'salesDesc' },
+  ];
+
+  const handleFilter = (value) => {
+    const newSearchParams = new URLSearchParams(searchParams);
+    newSearchParams.set('sortBy', value);
```

Comment:
Sort 컴포넌트에서 useSearchParams로 searchParams를 선언해놓으셨기때문에 newSearchParams는 굳이 새로 선언할 필요는 없어보입니다.
```suggestion
    searchParams.set('sortBy', value);
    setSearchParams(searchParams)
```
로 처리해주시면 될 것 같네요!